requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 flake8==3.5.*
 moto==2.0.10
-coverage==4.5.1
-codecov==2.0.15
+coverage==5.5
+codecov==2.1.11
```

Comment:
Coverage fixed a few bugs since we last updated it, so an upgrade was necessary to get things working with Python 3.7 again.